core/utils/dmljob.py

```diff
@@ -24,7 +24,8 @@ def __init__(
         uuid=None,
         raw_filepath=None,
         session_filepath=None,
-        datapoint_count=None
+        datapoint_count=None,
+        round_num=None,
```

Comment:
Could you explain what `round_num` is?

Reply:
Or actually can you put it in the docstring?

Reply:
Nice catch gotcha